### DISCUSSION TITLE
Restrict Environment Variables from being viewed

### DISCUSSION THREAD

**Discussion by nickjackson** [vercel/vercel#7691] [Help]
Hello,

Is there a way to prevent (certain) environment variables from being revealed in the UI and when running `vercel env pull`?

We are looking to up our security game internally and want to guard production secrets from being accessible without going through our internal process.

Thanks!

**Comment by nickjackson** [comment]
Feature on the way. https://twitter.com/rauchg/status/1534465811347283968

**Comment by n-sviridenko** [comment]
Any news?

**Comment by miguderp** [comment]
Bumping this as well instead of creating a new discussion
